src/NzbDrone.Core/Download/DownloadClientInfo.cs

```diff
@@ -8,10 +8,12 @@ public class DownloadClientInfo
         public DownloadClientInfo()
         {
             OutputRootFolders = new List<OsPath>();
+            HistoryRetention = true;
         }
 
         public bool IsLocalhost { get; set; }
         public string SortingMode { get; set; }
+        public bool HistoryRetention { get; set; }
```

Comment:
I think `RemovesCompletedDownloads`  would be more appropriate, which flips the logic a bit and the default will be false.

Reply:
reworked the entire check to RemovesCompletedDownloads, and also added qbit check in there.